src/App.jsx

```diff
@@ -14,30 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [message, setMessage] = useState('No goods selected');
+  const [selectedGood, setSelectedGood] = useState(null);
+  const [selectedIndex, setSelectedIndex] = useState(null);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {/* <h1 className="title is-flex is-align-items-center">No goods selected</h1> */}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <h1 className="title is-flex is-align-items-center">
+        {message}
+        <button
+          data-cy="ClearButton"
+          type="button"
+          className="delete ml-3"
+          onClick={() => {
+            setMessage(`No goods selected`);
+            setSelectedGood(null);
+            setSelectedIndex(null);
+          }}
+        />
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      <table className="table">
+        <tbody>
+          {goods.map((good, index) => {
```

Comment:
no needs to use `index`
```suggestion
          {goods.map((good) => {
```